common/locales/en/character.json

```diff
@@ -34,6 +34,7 @@
     "beard": "Beard",
     "mustache": "Mustache",
     "flower": "Flower",
+    "wheelChair": "Wheel Chair",
```

Comment:
I think this should be "Wheelchair" rather than "Wheel Chair". The former is the common spelling.